scripts/cob-command

```diff
@@ -0,0 +1,328 @@
+#!/bin/bash
+blue='\e[1;34m'
+black='\e[0;30m'
+red='\e[0;31m'
+green='\e[1;32m'
+
+standard_user="robot"
+
+ProgName=$(basename $0)
+
+get_clients_list(){
+  IP=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')
```

Comment:
is this entended to work on a local machine too? fails for me because I'm not using eth0